other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
Daemon creates the PID file manually. It appends its current PID to it, so the file will grow, and the daemon never deletes the PID file. I will make PID file optional in the future.